var/spack/repos/builtin/packages/py-dask-ml/package.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyDaskMl(Package):
+    """Scalable Machine Learning with Dask."""
+
+    homepage = "https://ml.dask.org/"
+    url      = "https://pypi.io/packages/source/d/dask-ml/dask-ml-1.8.0.tar.gz"
+
+    version('1.8.0', sha256='8fc4ac3ec1915e382fb8cae9ff1ec9b5ac1bee0b6f4c6975d6e6cb7191a4a815')
+
+    variant('docs', default=False, description='Build HTML documentation')
+    variant('xgboost', default=False, description='Deploys XGBoost alongside Dask')
+
+    extends('python')
```

Comment:
Once you subclass PythonPackage this line can be removed

Reply:
I removed it.